subgraphs/eigenlayer/src/mappings/handlers.ts

```diff
@@ -0,0 +1,345 @@
+import { Address, Bytes, ethereum, log } from "@graphprotocol/graph-ts";
+
+import {
+  ETH_ADDRESS,
+  INT_FOUR,
+  INT_THREE,
+  INT_TWO,
+  INT_ZERO,
+  PoolType,
+  WITHDRAWAL_QUEUED_DATA_TYPE,
+  WITHDRAWAL_QUEUED_SIGNATURE,
+  ZERO_ADDRESS,
+} from "../common/constants";
+import {
+  createPool,
+  getOrCreateAccount,
+  getOrCreateToken,
+  getPool,
+} from "../common/getters";
+import {
+  updatePoolIsActive,
+  updateTVL,
+  updateUsage,
+  updateVolume,
+} from "../common/metrics";
+import {
+  createDeposit,
+  createWithdraw,
+  getWithdraw,
+  updateWithdraw,
+} from "../common/events";
+import {
+  updateFinancialsDailySnapshot,
+  updatePoolDailySnapshot,
+  updatePoolHourlySnapshot,
+  updateUsageMetricsDailySnapshot,
+  updateUsageMetricsHourlySnapshot,
+} from "../common/snapshots";
+
+import { PodDeployed } from "../../generated/EigenPodManager/EigenPodManager";
+import { EigenPod } from "../../generated/EigenPodManager/EigenPod";
+import {
+  Deposit,
+  ShareWithdrawalQueued,
+  StrategyAddedToDepositWhitelist,
+  StrategyRemovedFromDepositWhitelist,
+  WithdrawalCompleted,
+} from "../../generated/StrategyManager/StrategyManager";
+import { Strategy } from "../../generated/StrategyManager/Strategy";
+
+/////////////////////////////////////////
+/////////// Native Restaking ////////////
+/////////////////////////////////////////
+
+export function handlePodDeployed(event: PodDeployed): void {
+  const podAddress = event.params.eigenPod;
+
+  // As per communication on EigenLayer's discord, currently native staking is technically not proven restaked
```

Comment:
If you can still find it, I would recommend pasting the link to the chat or thread here too